classes/class-dintero-checkout-gateway.php

```diff
@@ -65,7 +66,7 @@ function( $item, $cart_item_key ) {
 			 */
 			add_filter(
 				'woocommerce_hidden_order_itemmeta',
-				function( $hidden_meta ) {
+				function ( $hidden_meta ) {
 					$hidden_meta[] = '_dintero_checkout_line_id';
 					return $hidden_meta;
 				}
```

Comment:
Anonymous functions for actions should be avoided unless there is a good reason for it.